ui/app/AppLayouts/Chat/popups/community/TransferOwnershipPopup.qml

```diff
@@ -15,36 +16,40 @@ import shared 1.0
 StatusModal {
     id: root
 
-    property string privateKey
     property var store
+    property string privateKey
+    property string communityName
 
-    header.title: qsTr("Transfer ownership")
+    width: 640
+    height: 472
+    header.title: qsTr("Transfer ownership of ") + root.communityName + qsTr(" community")
     padding: 16
 
     ColumnLayout {
         anchors.fill: parent
-
         spacing: 16
 
+        StatusBaseText {
+            Layout.fillWidth: true
+            text: qsTr("I acknowledge that I lose ownership of this community by leaving and not saving the community key. I acknowledge that I lose ownership of this community by leaving and not saving the community key.")
```

Comment:
Why twice the same sentence?

Reply:
just what I copied from [here](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=3357%3A373868), removed duplicate :)

Reply:
Just show it once pls... :)